thread/src/main/kotlin/io/github/paulgriffith/kindling/thread/MultiThreadView.kt

```diff
@@ -51,7 +51,12 @@ class MultiThreadView(
     private val paths: List<Path>,
 ) : ToolPanel() {
     private val threadDumps = paths.map { path ->
-        ThreadDump.fromStream(path.inputStream()) ?: throw ToolOpeningException("Failed to open $path as a thread dump")
+        try {
+            ThreadDump.fromStream(path.inputStream())
+                ?: throw ToolOpeningException("Failed to open $path as a thread dump")
+        } catch (ex: Exception) {
+            throw ToolOpeningException("Failed to open $path as a thread dump", ex)
+        }
```

Comment:
I don't like these separated, identical throws, but couldn't come up with a better way to do this